core/log/metric/reader.go

```diff
@@ -94,7 +94,7 @@ func (r *defaultMetricLogReader) readMetricsInOneFile(filename string, offset ui
 	defer file.Close()
 
 	bufReader := bufio.NewReaderSize(file, 8192)
-	items := make([]*base.MetricItem, 0)
+	items := make([]*base.MetricItem, 0, 8)
```

Comment:
The metric file  default size is 1024 * 1024 * 50(50M);
From my perspective, we might need to set appropriate size for slice based on file size?

Reply:
Too many factors do affect, file offset, max lines, preSize.